src/test/java/nextstep/subway/acceptance/StationAcceptanceTest.java

```diff
@@ -36,52 +36,104 @@ public void setUp() {
     @DisplayName("지하철역을 생성한다.")
     @Test
     void createStation() {
-        // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
-
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
-                        .extract();
-
-        // then
-        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
-
-        // then
-        List<String> stationNames =
-                RestAssured.given().log().all()
-                        .when().get("/stations")
-                        .then().log().all()
-                        .extract().jsonPath().getList("name", String.class);
-        assertThat(stationNames).containsAnyOf("강남역");
+        // given: 지하철 역을 생성한다.
+        final String name = "강남역";
+
+        // when: 지하철 역을 생성한다.
+        ExtractableResponse<Response> response = createStationRequest(name);
+
+        // then: 지하철역이 정상적으로 생성되었고, 목록에 존재하는지 검증한다.
+        assertStatusCode(response, HttpStatus.CREATED);
+        assertThat(getStationNamesRequest()).containsAnyOf(name);
     }
 
     /**
      * Given 2개의 지하철역을 생성하고
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
     @DisplayName("지하철역을 조회한다.")
     @Test
     void getStations() {
+        // given: 2개의 지하철 역을 생성한다.
+        final String name1 = "선릉역";
+        createStationRequest(name1);
+
+        final String name2 = "역삼역";
+        createStationRequest(name2);
+
+        // when: 지하철 역들을 조회한다.
+        ExtractableResponse<Response> response = getStationsRequest();
+
+        // then: 지하철 역이 2개인지 검증한다.
+        final List<String> names = response.jsonPath()
+                .getList("name", String.class);
 
+        assertStatusCode(response, HttpStatus.OK);
+        assertThat(names.size()).isEqualTo(2);
```

Comment:
size만 검증하는 것은 생성된 지하철역이 응답되는 건지 정확하게 알 수 없을 것 같아요.
응답 받은 지하철역의 이름으로도 검증하면 어떨까요?

List size 검증은 이렇게도 할 수 있습니다. 😄
```suggestion
        assertThat(names).hasSize(2);
```

Reply:
@testrace 이 부분 저도 contains 검증까지 생각하긴 했는데, 테스트가 너무 세부까지 검증하는게 아닌가 싶어서 작성하지 않았는데요ㅎㅎ 
작성하는 것도 괜찮을 것 같습니다ㅎㅎ hasSize 감사합니다:)